pkgs/top-level/aliases.nix

```diff
@@ -576,6 +576,7 @@ mapAliases ({
   miopen-opencl = throw "'miopen-opencl' has been replaced with 'rocmPackages.miopen-opencl'"; # Added 2023-10-08
   mime-types = mailcap; # Added 2022-01-21
   minizip2 = pkgs.minizip-ng; # Added 2022-12-28
+  mobilecoin-wallet = throw "mobilecoin-wallet was removed from nixpkgs, because the repository does not exist anymore"; # Added 2023-11-09
```

Comment:
https://github.com/mobilecoinofficial/desktop-wallet still exists

Reply:
Changed the purpose of the PR to a version update